staging/src/k8s.io/apimachinery/pkg/runtime/codec.go

```diff
@@ -365,3 +365,8 @@ func (v multiGroupVersioner) KindForGroupVersionKinds(kinds []schema.GroupVersio
 	}
 	return schema.GroupVersionKind{}, false
 }
+
+// Identifier implements GroupVersioner interface.
+func (v multiGroupVersioner) Identifier() string {
+	return fmt.Sprintf("multi?target=%s&accepted=%s&coerce=%t", v.target, v.acceptedGroupKinds, v.coerce)
```

Comment:
maybe encode as a json map `{type:multi, target:..., accepted:..., coerce:...}`

Reply:
and switch the composed identifiers to do the same (nest children) (applies to encoder identifiers as well)

Reply:
done